test-common/src/main/java/com/mentoring/api/AbstractRequest.java

```diff
@@ -0,0 +1,135 @@
+package com.mentoring.api;
+
+import com.mentoring.allure.AllureAttachmentHandler;
+import com.mentoring.config.Config;
+import io.restassured.RestAssured;
+import io.restassured.http.Cookie;
+import io.restassured.http.Cookies;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import lombok.extern.slf4j.Slf4j;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+public abstract class AbstractRequest {
+
+    private static final String RESPONSE_BODY = "Response";
+    private static final String ERROR_RESPONSE = "Error response";
+
+    public Response sendPostRequest(Cookies cookies, String path, Object requestBody, int expectedStatusCode) {
```

Comment:
I saw that you call it multiple time with null first parameter. Here I would recommend to create and overloaded method which does not requires the cookies param.

Reply:
True, it looks much better now. Thanks!